sonar-html-plugin/src/main/resources/org/sonar/l10n/web/rules/Web/S6846.json

```diff
@@ -0,0 +1,24 @@
+{
+  "title": "DOM elements should not use the \"accesskey\" property",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "5min"
+  },
+  "tags": [
+    "accessibility"
+  ],
+  "defaultSeverity": "Minor",
+  "ruleSpecification": "RSPEC-6846",
+  "sqKey": "S6846",
+  "scope": "All",
+  "quickfix": "targeted",
```

Comment:
As mentioned in the RSPEC PR, the rule doesn't provide a quickfix. Let's not bother for now (unless you want to give it a try).